language/golang/build-images.md

```diff
@@ -372,10 +395,10 @@ images:
 ```console
 $ docker image ls
 
-REPOSITORY       TAG       IMAGE ID       CREATED          SIZE
-docker-gs-ping   latest    336a3f164d0f   43 minutes ago   540MB
-docker-gs-ping   v1.0      336a3f164d0f   43 minutes ago   540MB
-postgres         13.2      c5ec7353d87d   7 weeks ago      314MB
+REPOSITORY                       TAG       IMAGE ID       CREATED         SIZE
```

Comment:
```suggestion
REPOSITORY       TAG       IMAGE ID       CREATED          SIZE
```

I know none of these spacings are technically not correct or accurate, but generally elsewhere they're smaller to make the final rendered output a little nicer to look at.